src/commands/infra/validate.ts

```diff
@@ -157,10 +145,10 @@ export const validateEnvVariables = async (
           emoji.emojify(":no_entry_sign: " + i + " cannot be null.")
         );
       }
-      config.infra.checks.env_var_check = false;
+      config.infra!.checks!.env_var_check = false;
       return false;
     }
   }
-  config.infra.checks.env_var_check = true;
+  config.infra!.checks!.env_var_check = true;
   return true;
 };
```

Comment:
Discussion: In concept do we still need to have validate as a command?
`spk infra` is not executing terraform commands and checking the environment is handled by the pipeline.
Should validate instead be checking the tf version of sources before generating to see if the current environment is compatible. Here we can display warnings.

Reply:
this is because I have a helper function 
```
export const getConfig = (): IConfigYaml => {
  const config = Config();
  config.infra = config.infra || {};
  config.infra.checks = config.infra.checks || {};
  return config;
};
```
I will revert the change to not use the helper function.